recipes/vscode-jupyter/meta.yaml

```
@@ -0,0 +1,48 @@
+{% set name = "vscode-jupyter" %}
+{% set version = "2020.12.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version|replace("-", "_") }}
+
+source:
+  url: https://github.com/microsoft/{{ name }}/archive/{{ version }}.tar.gz
+  sha256: 99502db43f9f2ac4185d55ae51a70c81ffe9353983dcaca5cd1a7cf29f783a11
+
+build:
+  number: 0
+  skip: true  # [win]
+
+requirements:
+  build:
+    - nodejs 12.14.1
+    - code-server >=3.9
+    - {{ compiler('cxx') }}
+    - cmake
+  host:
+    - nodejs 12.14.1
+    - code-server >=3.9
+  run:
+    - code-server >=3.9
+    - python-jsonrpc-server ==0.4.0
+    - ujson ==3.2.0
```

Does this need to be so strict? 

Also omit `==` in recipes:

```suggestion
    - python-jsonrpc-server 0.4.0
    - ujson 3.2.0
```

---

The pins come from the [requirements.txt](https://github.com/microsoft/vscode-jupyter/blob/main/requirements.txt). But even in the [requirements.in](https://github.com/microsoft/vscode-jupyter/blob/main/requirements.in) `python-jsonrpc-server` is pinned to 0.4.0. The `ujson` pin is probably not necessary, as it is a third party dependency of `python-jsonrpc-server`.

Btw, in the requirements.in they also list a specific commit for a fork of `jedi-language-server`. The build has been working so far without that dependency, but it might be necessary during runtime. should we do about that? Just try the regular `jedi-language-server` as a dependency?

---

Removed `ujson` requirement in https://github.com/conda-forge/staged-recipes/pull/14117/commits/bce947a93ae6d8d27d3d3b055b2b1f91ce3aeafb.

---

Omitted `==` in https://github.com/conda-forge/staged-recipes/pull/14117/commits/3a26e5a6c24bb371877192825f4ce1e85be8addf.